boards/nucleo-f302r8/Makefile.features

```diff
@@ -6,6 +6,9 @@ FEATURES_PROVIDED += periph_spi
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
 
+# Put other features for this board (in alphabetical order)
```

Comment:
I'm not sure if we should add `riotboot` for these boards (the ones with 64kB of flash). Sure, you can flash a bootloader and split the flash in two slots but with only ~30k  per slot, you can't run `riotboot_flashwrite`, you can barely fit gnrc_minimal with 1.3kB of spare flash per slot.

Is there a use case for just `riotboot`? Should we enable a feature just cause we can?